lib/asset_cloud/base.rb

```diff
@@ -94,7 +96,7 @@ def initialize(root, url = '/')
     end
 
     def url_for(key, options={})
-      File.join(@url, URI.encode(key))
+      File.join(@url, Addressable::URI.encode(key))
```

Comment:
Could this be a breaking change? As in a URL that worked before stops working after this change.

Reply:
This is technically a breaking change since it returns different responses for the same input from previous versions.

Reply:
Ok. We should do some tests to see if it's going to break existing URLs.